frontend/src/layout/Sidebar.js

```diff
@@ -228,7 +228,7 @@ export function Sidebar({ user, sidebarCollapsed, setSidebarCollapsed }) {
                     </Menu.SubMenu>
                     <Menu.Item key="experiments" style={itemStyle} data-attr="menu-item-feature-f">
                         <ExperimentOutlined />
```

Comment:
We should maybe then consider switching up this icon? Although Antd's flag icons are pretty ugly

Reply:
Maybe ForkOutlined? @paolodamico thoughts?

Reply:
There's actually `FlagOutlined`, which could make sense?

Reply:
> Although Antd's flag icons are pretty ugly

That's my problem with it

Reply:
Depending on #1775 we might start introducing some custom icons for some stuff where default stuff isn't enough (e.g. retention, feature flags, ...), so I wouldn't worry to much about this right now.